src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -807,6 +811,7 @@ public ICraftingLink submitJob(final IGrid g, final ICraftingJob job, final Base
                         this.generateLinkData(craftID, requestingMachine == null, false),
                         this);
 
+                // No longer in use
                 this.prepareElapsedTime();
```

Comment:
Is this call necessary? If not, remove it and properly deprecate all related fields